examples/specs/vconcat_flatten.vl.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "Show a line chart corresponding to selected points.",
+  "data": {
+    "values": [
+      { "id": "001",
+        "ra": "243.35",
+        "dec": "+54.6",
+        "lc": [{"time": "1", "mag": 18.5}, {"time": "2", "mag": 19}]
+      },
+      { "id": "002",
+        "ra": "210.35",
+        "dec": "+14.6",
+        "lc": [{"time": "1", "mag": 19.5}, {"time": "2", "mag": 20}]
+      },
+      { "id": "003",
+        "ra": "143.35",
+        "dec": "+33.6",
+        "lc": [{"time": "1", "mag": 19}, {"time": "2", "mag": 18}]
+      }
+    ]
+  },
+  "transform": [
```

Comment:
Why transforming it up here?  Doing so require you to apply aggregate to basically unflatten it for the upper plot.